testsuite/replay_verify.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Aptos
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import subprocess
+import shutil
+from multiprocessing import Pool, freeze_support
+from typing import Tuple
+
+from verify_core.common import clear_artifacts, query_backup_latest_version
+
+# This script runs the replay-verify from the root of aptos-core
+# It assumes the aptos-backup-cli, replay-verify, and db-backup binaries are already built with the release profile
+
+
+def replay_verify_partition(
+    n: int,
+    N: int,
+    history_start: int,
+    per_partition: int,
+    latest_version: int,
+    txns_to_skip: Tuple[int],
+    backup_config_template_path: str,
+):
```

Comment:
you might wanna add return type annotations, I dont know if mypy operates this way but I recall that some typecheckers treat unannotated differently than annotated as returning None

Reply:
hm interesting. fixed!